joomla/joomla-cms
# Adds jlayout searchtools to com_users view levels

#### Description

This PR adds, for consistency, as it exists in so many other views in the backoffice, the search tools to com_users view levels.
###### Before PR

![image](https://cloud.githubusercontent.com/assets/9630530/12932993/af1c168a-cf7d-11e5-9e01-deb97d19ac44.png)
###### After PR

![image](https://cloud.githubusercontent.com/assets/9630530/12932971/8ee29fec-cf7d-11e5-8baa-0e394dd8018f.png)
#### How to test
1. Install latest staging and apply this patch
2. Go to Users -> Viewing Access Levels
3. Check the ordering select box and search is working fine.
